### ISSUE TITLE
The field `badges` in the JSON string is not defined in the `TopStoriesElement`

### ISSUE THREAD

**Issue by zhemaituk** [dataforseo/JavaClient#3]
Java client fails if API has new API fields.
Please disable such validation (or at least make it configurable) not to fail on unknown fields. Otherwise Java API client fails to be forward-compatible with potentially new API fields introduced in future.

Code to reproduce:

```
SerpGoogleOrganicLiveAdvancedRequestInfo request = new SerpGoogleOrganicLiveAdvancedRequestInfo();
request.setKeyword("microsoft");
request.setLocationCode(2840);
request.setLanguageCode("en");
request.setDevice("desktop");
request.setOs("windows");
request.setDepth(limit);

SerpGoogleOrganicLiveAdvancedResponseInfo response = serpApi.googleOrganicLiveAdvanced(List.of(request));
```

Stacktrace:
```
java.lang.IllegalArgumentException: The field `badges` in the JSON string is not defined in the `TopStoriesElement` properties. JSON: {"type":"top_stories_element","source":"Microsoft","domain":"www.microsoft.com","title":"Register for Research Forum, Arena learning, MInference, HyperNova","date":"2 hours ago","amp_version":false,"timestamp":"2024-08-28 16:08:06 +00:00","url":"https://www.microsoft.com/en-us/research/blog/research-focus-week-of-august-26-2024/","image_url":"https://api.dataforseo.com/cdn/i/08282108-6748-0139-0000-4259106731a1:0","badges":null}
	at org.dataforseo.client.model.TopStoriesElement.validateJsonElement(TopStoriesElement.java:368) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.model.TopStoriesElement$CustomTypeAdapterFactory$1.read(TopStoriesElement.java:419) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.model.TopStoriesElement$CustomTypeAdapterFactory$1.read(TopStoriesElement.java:409) ~[dataforseo-client-1.0.12.jar:na]
	at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:204) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:40) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.readIntoField(ReflectiveTypeAdapterFactory.java:212) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$FieldReflectionAdapter.readField(ReflectiveTypeAdapterFactory.java:433) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:393) ~[gson-2.10.1.jar:na]
	at io.gsonfire.util.JsonUtils$ConfigurableTypeAdapter.read(JsonUtils.java:99) ~[gson-fire-1.9.0.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at io.gsonfire.util.JsonUtils.fromJsonTree(JsonUtils.java:72) ~[gson-fire-1.9.0.jar:na]
	at io.gsonfire.gson.HooksTypeAdapter.read(HooksTypeAdapter.java:53) ~[gson-fire-1.9.0.jar:na]
	at io.gsonfire.util.JsonUtils$ConfigurableTypeAdapter.read(JsonUtils.java:99) ~[gson-fire-1.9.0.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at io.gsonfire.util.JsonUtils.fromJsonTree(JsonUtils.java:72) ~[gson-fire-1.9.0.jar:na]
	at io.gsonfire.gson.HooksTypeAdapter.read(HooksTypeAdapter.java:53) ~[gson-fire-1.9.0.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at org.dataforseo.client.model.TopStoriesSerpElementItem$CustomTypeAdapterFactory$1.read(TopStoriesSerpElementItem.java:360) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.model.TopStoriesSerpElementItem$CustomTypeAdapterFactory$1.read(TopStoriesSerpElementItem.java:349) ~[dataforseo-client-1.0.12.jar:na]
	at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:204) ~[gson-2.10.1.jar:na]
	at io.gsonfire.util.JsonUtils$ConfigurableTypeAdapter.read(JsonUtils.java:99) ~[gson-fire-1.9.0.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at io.gsonfire.util.JsonUtils.fromJsonTree(JsonUtils.java:72) ~[gson-fire-1.9.0.jar:na]
	at io.gsonfire.gson.TypeSelectorTypeAdapterFactory$TypeSelectorTypeAdapter.read(TypeSelectorTypeAdapterFactory.java:82) ~[gson-fire-1.9.0.jar:na]
	at io.gsonfire.gson.NullableTypeAdapter.read(NullableTypeAdapter.java:36) ~[gson-fire-1.9.0.jar:na]
	at io.gsonfire.util.JsonUtils$ConfigurableTypeAdapter.read(JsonUtils.java:99) ~[gson-fire-1.9.0.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at io.gsonfire.util.JsonUtils.fromJsonTree(JsonUtils.java:72) ~[gson-fire-1.9.0.jar:na]
	at io.gsonfire.gson.HooksTypeAdapter.read(HooksTypeAdapter.java:53) ~[gson-fire-1.9.0.jar:na]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:40) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.readIntoField(ReflectiveTypeAdapterFactory.java:212) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$FieldReflectionAdapter.readField(ReflectiveTypeAdapterFactory.java:433) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:393) ~[gson-2.10.1.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at org.dataforseo.client.model.SerpGoogleOrganicLiveAdvancedResultInfo$CustomTypeAdapterFactory$1.read(SerpGoogleOrganicLiveAdvancedResultInfo.java:534) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.model.SerpGoogleOrganicLiveAdvancedResultInfo$CustomTypeAdapterFactory$1.read(SerpGoogleOrganicLiveAdvancedResultInfo.java:523) ~[dataforseo-client-1.0.12.jar:na]
	at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:204) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:40) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.readIntoField(ReflectiveTypeAdapterFactory.java:212) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$FieldReflectionAdapter.readField(ReflectiveTypeAdapterFactory.java:433) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:393) ~[gson-2.10.1.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at org.dataforseo.client.model.SerpGoogleOrganicLiveAdvancedTaskInfo$CustomTypeAdapterFactory$1.read(SerpGoogleOrganicLiveAdvancedTaskInfo.java:442) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.model.SerpGoogleOrganicLiveAdvancedTaskInfo$CustomTypeAdapterFactory$1.read(SerpGoogleOrganicLiveAdvancedTaskInfo.java:431) ~[dataforseo-client-1.0.12.jar:na]
	at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:204) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:40) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.readIntoField(ReflectiveTypeAdapterFactory.java:212) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$FieldReflectionAdapter.readField(ReflectiveTypeAdapterFactory.java:433) ~[gson-2.10.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:393) ~[gson-2.10.1.jar:na]
	at com.google.gson.TypeAdapter.fromJsonTree(TypeAdapter.java:299) ~[gson-2.10.1.jar:na]
	at org.dataforseo.client.model.SerpGoogleOrganicLiveAdvancedResponseInfo$CustomTypeAdapterFactory$1.read(SerpGoogleOrganicLiveAdvancedResponseInfo.java:404) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.model.SerpGoogleOrganicLiveAdvancedResponseInfo$CustomTypeAdapterFactory$1.read(SerpGoogleOrganicLiveAdvancedResponseInfo.java:393) ~[dataforseo-client-1.0.12.jar:na]
	at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:204) ~[gson-2.10.1.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:1137) ~[gson-2.10.1.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:1047) ~[gson-2.10.1.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:1014) ~[gson-2.10.1.jar:na]
	at org.dataforseo.client.JSON.deserialize(JSON.java:4121) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.ApiClient.deserialize(ApiClient.java:902) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.ApiClient.handleResponse(ApiClient.java:1112) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.ApiClient.execute(ApiClient.java:1036) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.api.SerpApi.googleOrganicLiveAdvancedWithHttpInfo(SerpApi.java:9060) ~[dataforseo-client-1.0.12.jar:na]
	at org.dataforseo.client.api.SerpApi.googleOrganicLiveAdvanced(SerpApi.java:9041) ~[dataforseo-client-1.0.12.jar:na]
```

**Comment by lVics** [comment]
Issue fixed in latest version (1.0.16)

**Comment by zhemaituk** [comment]
Thank you! Confirming it works well now
